.changeset/chilly-terms-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-module-stack-overflow-collator': patch
```

Comment:
```suggestion
'@backstage/plugin-search-backend-module-stack-overflow-collator': minor
```

To have it go to `0.1.0` for the initial release

Reply:
Thanks, done here: https://github.com/backstage/backstage/pull/20456/commits/6cd61d379d47cc46592c49a761abf7d325e448b4 🙏🏻